tests/reporter/kubernetes.go

```diff
@@ -289,7 +289,7 @@ func (r *KubernetesReporter) logDMESG(virtCli kubecli.KubevirtClient, logsdir st
 			commands := []string{
 				virt_chroot.GetChrootBinaryPath(),
 				"--mount",
-				virt_chroot.GetChrootMountNamespace(),
+				virt_chroot.MountNamespace,
```

Comment:
If we drop it from `virt-chroot`, `MountNamespace` constant is better redefined here.

Reply:
@vasiliy-ul  patched in.